src/oops/SOLID/singleResponsibilityPrinciple/before/Employee.java

```diff
@@ -12,6 +12,15 @@
     private String firstName;
     private String lastName;
     private int monthlyIncome;
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
```

Comment:
You could have reused getFullName() instead of adding new methods here. Logic is the same.